OpenRA.Mods.Common/Traits/SupportPowers/NukePower.cs

```diff
@@ -134,9 +134,14 @@ public override void Activate(Actor self, Order order, SupportPowerManager manag
 				var beacon = new Beacon(
 					order.Player,
 					targetPosition,
-					Info.BeaconPalettePrefix,
+					Info.BeaconPaletteIsPlayerPalette,
+					Info.BeaconPalette,
+					Info.BeaconImage,
 					Info.BeaconPoster,
 					Info.BeaconPosterPalette,
+					Info.CircleSequence,
+					Info.ArrowSequence,
```

Comment:
You've got the order of the arguments flipped around here: `Info.ArrowSequence` comes before `Info.CircleSequence`.